studio/src/lib/schema-helpers.ts

```diff
@@ -57,7 +58,7 @@ export type GraphQLTypeCategory =
 export type ParsedGraphQLField = {
   name: string;
   description?: string;
-  deprecationReason?: string;
+  deprecationReason?: string | null;
```

Comment:
no `?`, right?

Reply:
I'm leaving it optional as the fields for the `union` type doesn't seems to care about this